ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -669,53 +631,24 @@ fi
 # (Also, 'fix -a' is hard-coded to use '.fix' as the list of noise components, although that 
 # could be worked around).
 
-export FSL_FIX_WBC="${Caret7_Command}"
```

Comment:
Is this getting set somewhere else?

Reply:
Our version of `fix_3_clean` has the getenvs commented out, passes `WBC` as a function argument, and no longer makes internal modifications to the matlab path.

Reply:
More to the point, we no longer source legacy fix `settings.sh` in `ReApplyFixMultiRun...` anyway, so none of the extra `FSL_FIX` stuff gets set, because we don't use any unmodified legacy fix stuff.